ocaml/database/db_connections.ml

```diff
@@ -102,9 +103,10 @@ let flush_dirty_and_maybe_exit dbconn exit_spec =
 
 let flush dbconn db =
   debug "About to flush database: %s" dbconn.Parse_db_conf.path;
+  let fsync = dbconn.fsync_enabled in
```

Comment:
Since this variable is only used once you can pass it directly like this:
`Backend_xml.flush dbconn db ~fsync:(dbconn.Db_cache_types.fsync_enabled)`